registrator.go

```diff
@@ -172,6 +173,10 @@ func main() {
 			go b.Add(msg.ID)
 		case "die":
 			go b.RemoveOnExit(msg.ID)
+		case "kill":
+			if *deregisterOnStop {
+				go b.RemoveOnExit(msg.ID)
+			}
 		}
```

Comment:
Oh I see, @mattaitchison. So it sounds like this could still be a good addition as an option, but it shouldn't be made the default behavior. Running `docker kill -s=SIGHUP` is a bit of a corner case, and I would imagine that the way many people run containers with registrator (especially inside orchestration frameworks), that this isn't something we'd need to worry about 90% of the time.

@mattaitchison, given that this option could be very useful to people who aren't sending arbitrary signals to their containers via `docker kill`, which is likely the majority case, do you think this is still a valid feature request?